doc/sphinxman/source/bibliography.rst

```diff
@@ -746,3 +746,7 @@ Bibliography
 .. not yet referenced    *J. Chem. Phys.* **137**, 224106 (2012).
 .. not yet referenced    https://doi.org/10.1063/1.4768233
 
+.. [Matthews:2020:1382]
+   D. A. Matthews
+   *J. Chem. Theory Comput.* **16**, 1382 (2020).
+   https://doi.org/10.1021/acs.jctc.9b01205
```

Comment:
Adding a reference is good, but the sphinx build breaks (we have it on strict) if you don't use the reference somewhere in the narrative docs. Please actually use the ref or add comments like the previous entry's fix.

Reply:
Added the `not yet referenced` keywords!

Reply:
Just FYI, "not yet referenced" isn't doing anything -- `.. anything` is the very weird comment marker for restructured text.